.github/workflows/specification-version-check.yml

```diff
@@ -0,0 +1,27 @@
+on:
+  schedule:
+    - cron: "0 13 * * *"
+  workflow_dispatch:
+name: Specification version check
+jobs:
+  # Get the version of the TUF specification the project states it supports
+  get-supported-tuf-version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.get-version.outputs.version }}
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+      - uses: actions/setup-python@98f2ad02fd48d057ee3b4d4f66525b231c3e52b6
+      - id: get-version
+        run: |
+          python3 -m pip install -e .
+          script="from tuf.api.metadata import SPECIFICATION_VERSION; \
+                  print(f\"v{'.'.join(SPECIFICATION_VERSION)}\")"
+          ver=$(python3 -c "$script")
+          echo "::set-output name=version::$ver"
+  # Get the latest TUF specification release and open an issue (if needed)
+  specification-bump-check:
+    needs: get-supported-tuf-version
+    uses: theupdateframework/specification/.github/workflows/check-latest-spec-version.yml@master
```

Comment:
this does bring to surface a small issue with this approach: We do not want `@master` in our workflows... but on the other hand, getting dependabot updates to workflows on every spec commit sounds bad (I assume that's how it's going to work).

Hmm...

Reply:
It's a possible drawback, indeed. Unfortunately, I think the only way to change that is to bundle it as a Github Action and create a separate repository just for it.

Reply:
In that sense, I'll update the PR and add permissions to the job calling the workflow in order to protect the project from any vulnerabilities coming from the `specification` repo 👍